### PR TITLE
rewrite introduction to verify section

### LINKED ISSUE #86: refactor "Test/Overview" page

### PR DESCRIPTION
**Pull request by kevinsung**
Fixes #86

### CODE REVIEW

**Comment by kevinsung** [comment]
In the preview, the upper bar still says "Test" instead of "Verify" and is a broken link. How do I fix that?
image

**Comment by abbycross** [comment]
FYI @Eric-Arellano 
It's a known problem and I think (??) we need the assistance of the frontend devs but Eric can confirm. We also need to remove "Optimize" and very soon add "Transpile" (also a known to-do)

File: docs/verify/index.mdx [hunk 1]
```diff
@@ -4,9 +4,18 @@ description: Introduction to the Verify phase
...
+
+The following tools are available for you to verify your quantum programs:
+
+- For exact simulation of small quantum circuits, you can use the reference primitives included with Qiskit. See [Exact simulation with Qiskit primitives](simulate-with-qiskit-primitives).
+- For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer).
+- To build your own custom noise models, use the [`noise`](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html) module of Qiskit Aer. See [Building noise models](building_noise_models).
+- As mentioned before, Qiskit Aer can be used to efficiently simulate stabilizer circuits, a special restricted class of quantum circuits. For example, using Qiskit Aer, you can simulate stabilizer circuits with thousands of qubits. See [Efficient simulation of stabilizer circuits with Qiskit Aer primitives](stabilizer-circuit-simulation).
```
Comment (javabster) [inline docs/verify/index.mdx:21]: feels a bit repetitive to have this bullet point again about extended stabilisers. Maybe instead adjust the sentence before the bullet points to say "in addition, the following tools are available..."

Comment (kevinsung) [reply docs/verify/index.mdx:21]: Done.

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Review by abbycross — APPROVED** [review]

**Pull request merged**